front-end/src/ph/calibration_wizard.jsx

```diff
@@ -29,34 +30,53 @@ export default class CalibrationWizard extends React.Component {
 
   componentDidMount () {
     this.promise = new $.Deferred()
+    this.timer = setInterval(() => {
+      this.readProbe()
+    }, 500)
   }
 
-  calibrate (point, value) {
+  componentWillUnmount () {
+    window.clearInterval(this.timer)
+  }
+
+  readProbe () {
+    let id = this.props.probe.id
+    let that = this
+
+    fetch('/api/phprobes/' + id + '/read', {
```

Comment:
api calls like this does not need to go through the redux layer?

Reply:
I was wondering about that too.  I didn't put it in redux since it didn't really need to be part of application state.  I'll refactor to put it in redux and state.  It would be more consistent that way, and maybe it would be helpful if we changed it to use a rolling average over a small window.

I'll address this within the next few days.